views/projects/better_egovernment.erb

```diff
@@ -8,8 +8,9 @@
           <li><a href="/projekty/red-flags">Red Flags</a></li>
           <li class="sidebar-menu-item-active">Lepší e-Government</li>
           <li><a href="/projekty/ekosystem">Ekosystém</a></li>
-          <li><a href="https://navody.digital">Návody.Digital</a></li>
+          <li><a href="https://navody.digital" target="_blank">Návody.Digital</a></li>
```

Comment:
_blank je bad practice.

Reply:
Aj ked link vedie externe?

Reply:
Ano, lebo nenechavas toto rozhodnutie na pouzivatela a navyse to urobi nieco co mozno ani necaka. Su nejake vynimky ked to chces urobit (napr login screen v externej sluzbe), ale je to extremne zriedkave.

Reply:
Oki, rozumiem. Teraz mimo toho otazka, ked mas napriklad favor link, na inu organizaciu/domenu, v clanku napriklad, tiez same tab?

Reply:
Co je to favor link? Ja som v tomto extremista, lebo mna osobne toto nasiera ked mi to weby robia. Ale rozumiem, ze si niekto moze **mysliet**, ze ked to otvori do _blank tak mu z webu neodide nejaka cast ludi. Cisla k tomu som vsak nikdy nevidel. Je to podla mna skor mytus.

Reply:
Rozumiem. Favor link, ked napriklad pisu v dennik N o SD a pridaju link na niaky nas projekt alebo sluzbu ktoru odporucaju alebo zmienuju.